scripts/module-map.js

```diff
@@ -8,7 +8,7 @@
 module.exports = Object.assign(
   {
     immutable: 'immutable',
-    React: 'react',
```

Comment:
Can you elaborate more on change?

Reply:
Sure, I updated the commit message in https://github.com/twreporter/draft-js/pull/1/commits/862ad3e28b118c38fa46c031669284ad5b01c808.

Please have a see.

Reply:
OK, I supposed that that's why the CI of the original repo is broken.